### DISCUSSION TITLE
zksync compile: Contracts folder not found in the workspace

### DISCUSSION THREAD

**Discussion by dannyakorede12** [Cyfrin/foundry-full-course-cu#2393] [Q&A]
Hey, I just activated the zksync plugin and I'm unable to compile, it says "Contracts folder not found in the workspace" I tried to create the contract folder manually on the remix workspace and i copied the Simple Storage file into this Contracts folder but that didn't seem to work either.

Please help

**Comment by dannyakorede12** [comment]
a "caps-lock" error when naming the contracts folder

**Reply by EngrPips** [reply]
Hey @dannyakorede12, It seems like you resolved this issue already.

**Reply by EngrPips** [reply]
> a "caps-lock" error when naming the contracts folder

Did you check this @Danities316

**Reply by HasanZaigam** [reply]
Yahhh! It is a **case sensitivity problem**. When I write the file name as **FirstSolContract.sol,** I receive an error message. However, when I change the file name to **first_sol_contract.sol**, there is no error.

@Danities316 @EngrPips

**Reply by EngrPips** [reply]
Interesting.

**Comment by Danities316** [comment]
@EngrPips Iam currently getting same issues

**Reply by ghost** [reply]
Put the contract file in a file folder named "contracts" (the "c" is lowercase)

**Comment by EngrPips** [comment]
Great Heads-up

**Comment by brlebtag** [comment]
I am having the same issue. I already created a "contracts" folder but no luck! I also rename the file to SImpleStorage.sol to simple_storage.sol and nothing. Any idea? I am using zksync v1.5.6

**Reply by EngrPips** [reply]
You can maybe show the exact error you are getting, your code, and your working directory, as those may help us catch the issue.
